packages/quantic/playwright/page-object/configurationObject.ts

```diff
@@ -6,19 +6,19 @@ export class ConfigurationObject {
   }
 
   get tryButton(): Locator {
-    return this.page.locator('lightning-button[data-cy="cfg-try"]');
+    return this.page.locator('lightning-button[data-testid="cfg-try"]');
```

Comment:
I think these ones should instead be updated to:


```suggestion
    return this.page.getByTestId('cfg-try');
```